css/global.css

```diff
@@ -24,6 +24,13 @@
   display: none !important;
 }
 
+/* also hides applicant count at top of page at linkedin.com/jobs/collections when it's not in bold */
+
+.job-details-jobs-unified-top-card__primary-description-without-tagline:has(
```

Comment:
Looks for an element with the class name `job-details-jobs-unified-top-card__primary-description-without-tagline` that has a child element with the class name `tvm__text--neutral` and selects that elements last child element. That element is then removed from the DOM.